src/Video.tsx

```diff
@@ -615,15 +611,14 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
           drm={_drm}
           style={StyleSheet.absoluteFill}
           resizeMode={resizeMode}
-          fullscreen={isFullscreen}
```

Comment:
This PR is big, can you confirm the fullscreen property is still available for apps ?

Reply:
previously, we use internal fullscreen state(for handle present,dismiss fullscreen command) but it was meaningless.
fullscreen prop will be inject with `{...rest}` and fullscreen commands are work with command.setFullscreen(value).
previous
Rather, the old code was causing different behavior than expected. when user changing fullscreen prop value.